sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -159,7 +159,7 @@ Please note that models can also be built using a graphical user interface such
 
 Because analyzing documents and building models take time, these operations are implemented as [**long-running operations**][dotnet_lro_guidelines].  Long-running operations consist of an initial request sent to the service to start an operation, followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has succeeded, to get the result.
 
-For long running operations in the Azure SDK, the client exposes a `Start<operation-name>` method that returns an `Operation<T>`.  You can use the extension method `WaitForCompletionAsync()` to wait for the operation to complete and obtain its result.  A sample code snippet is provided to illustrate using long-running operations [below](#extract-layout).
+For long running operations in the Azure SDK, the client exposes a method that returns an `Operation<T>` object. You can set its parameter `waitUntil` to `WaitUntil.Completed` to wait for the operation to complete and obtain its result; or set it to `WaitUntil.Started` if you just want to start the operation and consume the result later. A sample code snippet is provided to illustrate using long-running operations [below](#extract-layout).
```

Comment:
Is this a new pattern we have created for LRO? If so, how is the pattern in FA different from other LRO patterns? Is this common across cognitive space so say TA would benefit from this too? 

Also, do we need to publicize this pattern in our design guidelines?

Reply:
Looks like this is an existing pattern. WaitUntil is a type that already exists :)

Reply:
That's right. It's a new pattern architects want libraries to follow from now on. From my understanding, libraries that already implement the old pattern should keep doing so for consistency.

The pattern is not described in our guidelines yet. Krzysztof said we should remind him to update it.